README.md

```diff
@@ -86,23 +92,28 @@ Command definitions:
                 }
             }
         }.
+```
 
-Calculator provides "sum" command that prints a sum of integer numbers:
+The calculator provides "sum" command that prints a sum of integer numbers:
 
+```shell
     $ ./calc sum 1 2 3
     6
+```
 
 Math sub-commands provide trigonometric functions:
 
+```shell
     $ ./calc math cos 1.4
     0.16996714290024104
     $ ./calc math sin 1.4
     0.9854497299884601
+```
 
 ## Complex applications
 
 CLI framework is capable of handling releases containing hundreds of modules
-implementing cli behaviour. Commands may be exported from multiple modules and
+implementing cli behavior. Commands may be exported from multiple modules and
```

Comment:
Could you please keep British spelling of behaviour?

Reply:
Ofc I will change it. I use Grammarly VScode plugin and it suggested the change.